src/lib/crypto/bn.c

```diff
@@ -221,15 +221,15 @@ bn_bn2hex(const PGPV_BIGNUM *a)
     return NULL;
 }
 
-/* hash a bignum, possibly padded - first length, then string itself */
+/* hash a bignum_t, possibly padded - first length, then string itself */
 size_t
-bn_hash(const PGPV_BIGNUM *bignum, pgp_hash_t *hash)
+bn_hash(const bignum_t *bignum_t, pgp_hash_t *hash)
```

Comment:
I don't think this was intentional.

Reply:
Thnx. Wasn't. closing this as automatic sed wasn't succcessful